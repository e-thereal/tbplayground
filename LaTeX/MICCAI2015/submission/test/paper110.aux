\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\citation{souplet2008}
\citation{weiss2013}
\citation{geremia2010}
\citation{yoo2014}
\citation{Ciresan2012}
\citation{LeCun1998}
\@writefile{toc}{\contentsline {title}{Deep Convolutional Encoder Networks for Multiple Sclerosis Lesion Segmentation}{1}{chapter.1}}
\@writefile{toc}{\authcount {6}}
\@writefile{toc}{\contentsline {author}{Tom Brosch\unskip {} \and Youngjin Yoo\unskip {} \and Lisa Y.W. Tang\unskip {} \and David K.B. Li\unskip {} \and \unhbox \voidb@x \hbox {Anthony Traboulsee}\unskip {} \and Roger Tam\unskip {} }{1}{chapter.1}}
\@writefile{toc}{\contentsline {section}{\numberline {1}Introduction}{1}{section.1.1}}
\citation{kang2014fully}
\citation{LeCun1998}
\citation{zeiler2011}
\citation{masci2011}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces Convolutional encoder network used to produce a lesion segmentation, $S$, from multi-modal images, $I = (I_\text  {FLAIR}, I_\text  {T1}, I_\text  {T2})$. The first two layers form a convolutional neural network with trainable filter kernels $w^{(1)}_{ij}$, and the last two layers form a deconvolutional neural network with trainable filter kernels $w^{(2)}_j$.}}{3}{figure.1.1}}
\newlabel{fig:network}{{1}{3}{Convolutional encoder network used to produce a lesion segmentation, $S$, from multi-modal images, $I = (I_\text {FLAIR}, I_\text {T1}, I_\text {T2})$. The first two layers form a convolutional neural network with trainable filter kernels $w^{(1)}_{ij}$, and the last two layers form a deconvolutional neural network with trainable filter kernels $w^{(2)}_j$}{figure.1.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Methods}{3}{section.1.2}}
\newlabel{sec:method}{{2}{3}{Methods}{section.1.2}{}}
\newlabel{eq:segprob}{{1}{3}{Methods}{equation.1.2.1}{}}
\citation{LeCun1998}
\newlabel{eq:dE2}{{5}{4}{Methods}{equation.1.2.5}{}}
\newlabel{eq:delta2}{{6}{4}{Methods}{equation.1.2.6}{}}
\newlabel{eq:dE1}{{7}{4}{Methods}{equation.1.2.7}{}}
\newlabel{eq:delta1}{{8}{4}{Methods}{equation.1.2.8}{}}
\citation{styner20083d}
\citation{Ciresan2012}
\citation{souplet2008}
\citation{weiss2013}
\citation{geremia2010}
\@writefile{toc}{\contentsline {section}{\numberline {3}Experiments and Results}{5}{section.1.3}}
\@writefile{lof}{\contentsline {figure}{\numberline {2}{\ignorespaces Example segmentations of our method for three different subjects from the challenge data set. Our method performed well and consistently despite the large contrast differences seen between the first two rows. In the third row, our method also segmented lesions that have similar contrast, but these regions had not been identified as lesions by the manual rater, which highlights the difficulty in distinguishing focal lesions from diffuse damage, even for experts.}}{6}{figure.1.2}}
\newlabel{fig:segmentation}{{2}{6}{Example segmentations of our method for three different subjects from the challenge data set. Our method performed well and consistently despite the large contrast differences seen between the first two rows. In the third row, our method also segmented lesions that have similar contrast, but these regions had not been identified as lesions by the manual rater, which highlights the difficulty in distinguishing focal lesions from diffuse damage, even for experts}{figure.1.2}{}}
\citation{souplet2008}
\citation{weiss2013}
\citation{geremia2010}
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces Comparison of our method with state-of-the-art lesion segmentation methods in terms of mean TPR, PPV, and DSC. Our method performs comparably to the best methods reported on the MS lesion segmentation challenge data set.}}{7}{table.1.1}}
\newlabel{tab:state}{{1}{7}{Comparison of our method with state-of-the-art lesion segmentation methods in terms of mean TPR, PPV, and DSC. Our method performs comparably to the best methods reported on the MS lesion segmentation challenge data set}{table.1.1}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {3}{\ignorespaces Comparison of DSC scores calculated on the training and test sets for varying numbers of training samples. At around 100 samples, the model becomes stable in terms of test performance and the small difference between training and test DSCs, indicating that overfitting of the training data no longer occurs.}}{7}{figure.1.3}}
\newlabel{fig:bioms}{{3}{7}{Comparison of DSC scores calculated on the training and test sets for varying numbers of training samples. At around 100 samples, the model becomes stable in terms of test performance and the small difference between training and test DSCs, indicating that overfitting of the training data no longer occurs}{figure.1.3}{}}
\@writefile{toc}{\contentsline {section}{\numberline {4}Conclusions}{7}{section.1.4}}
\bibstyle{splncs}
\bibdata{bib}
\bibcite{souplet2008}{1}
\bibcite{weiss2013}{2}
\bibcite{geremia2010}{3}
\bibcite{yoo2014}{4}
\bibcite{Ciresan2012}{5}
\bibcite{LeCun1998}{6}
\bibcite{kang2014fully}{7}
\bibcite{zeiler2011}{8}
\bibcite{masci2011}{9}
\bibcite{styner20083d}{10}
\newlabel{sec:achnowledge}{{4}{8}{Acknowledgements}{section*.2}{}}
