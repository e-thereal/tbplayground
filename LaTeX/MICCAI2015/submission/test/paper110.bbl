\begin{thebibliography}{10}

\bibitem{souplet2008}
Souplet, J.C., Lebrun, C., Ayache, N., Malandain, G.:
\newblock An automatic segmentation of {T2-FLAIR} multiple sclerosis lesions.
\newblock In: MIDAS Journal - MICCAI 2008 Workshop. (2008)

\bibitem{weiss2013}
Weiss, N., Rueckert, D., Rao, A.:
\newblock Multiple sclerosis lesion segmentation using dictionary learning and
  sparse coding.
\newblock In: Mori, K., Sakuma, I., Sato, Y., Barillot, C., Navab, N. (eds.)
  MICCAI 2013, Part I. LNCS 8149.
\newblock Springer, Heidelberg (2013)  735--742

\bibitem{geremia2010}
Geremia, E., Menze, B.H., Clatz, O., Konukoglu, E., Criminisi, A., Ayache, N.:
\newblock Spatial decision forests for {MS} lesion segmentation in
  multi-channel {MR} images.
\newblock In: Jian, T., Navab, N., Pluim, J., Viergever, M. (eds.) MICCAI 2010,
  Part I. LNCS, vol. 6362, Springer, Heidelberg (2010)  111--118

\bibitem{yoo2014}
Yoo, Y., Brosch, T., Traboulsee, A., Li, D.K., Tam, R.:
\newblock Deep learning of image features from unlabeled data for multiple
  sclerosis lesion segmentation.
\newblock In: Wu, G., Zhang D., Zhou L. (eds.) MLMI 2014, LNCS, vol. 8679.
\newblock Springer, Heidelberg (2014)  117--124

\bibitem{Ciresan2012}
Ciresan, D., Giusti, A., Schmidhuber, J.:
\newblock Deep neural networks segment neuronal membranes in electron
  microscopy images.
\newblock Advances in Neural Information Processing Systems (2012)  1--9

\bibitem{LeCun1998}
LeCun, Y., Bottou, L., Bengio, Y., Haffner, P.:
\newblock Gradient-based learning applied to document recognition.
\newblock Proceedings of the {IEEE} \textbf{86}(11) (1998)  2278--2324

\bibitem{kang2014fully}
Kang, K., Wang, X.:
\newblock Fully convolutional neural networks for crowd segmentation.
\newblock arXiv preprint arXiv:1411.4464 (2014)

\bibitem{zeiler2011}
Zeiler, M.D., Taylor, G.W., Fergus, R.:
\newblock Adaptive deconvolutional networks for mid and high level feature
  learning.
\newblock In: 2011 IEEE International Conference on Computer Vision (ICCV),
  IEEE (2011)  2018--2025

\bibitem{masci2011}
Masci, J., Meier, U., Cire{\c{s}}an, D., Schmidhuber, J.:
\newblock Stacked convolutional auto-encoders for hierarchical feature
  extraction.
\newblock In: Artificial Neural Networks and Machine Learning--ICANN 2011.
\newblock Springer (2011)  52--59

\bibitem{styner20083d}
Styner, M., Lee, J., Chin, B., Chin, M., Commowick, O., Tran, H.,
  Markovic-Plese, S., Jewells, V., Warfield, S.:
\newblock {3D} segmentation in the clinic: A grand challenge {II}: {MS} lesion
  segmentation.
\newblock MIDAS Journal - MICCAI 2008 Workshop (2008)  1--6

\end{thebibliography}
