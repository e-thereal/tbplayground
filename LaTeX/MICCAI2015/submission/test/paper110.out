\BOOKMARK [0][-]{chapter.1}{Deep Convolutional Encoder Networks for Multiple Sclerosis Lesion Segmentation}{}% 1
